src/createSource.mts

```diff
@@ -167,5 +192,9 @@ export const createSource = async ({
       name: "suspense.ts",
       content: suspenseResult,
     },
+    {
+      name: "prefetch.ts",
```

Comment:
Would it be possible to use `OpenApiRqFiles` from constants.mts for this part as well?